### PR TITLE
[zh] Sync 1.6 release note into Chinese

### PR DESCRIPTION
**Pull request by wilsonwu**
Sync 1.6 release note into Chinese:
```
content\zh\news\releases\1.6.x\announcing-1.6\_index.md
```

- [ ] Ambient
- [x] Docs
- [ ] Installation
- [ ] Networking
- [ ] Performance and Scalability
- [ ] Extensions and Telemetry
- [ ] Security
- [ ] Test and Release
- [ ] User Experience
- [ ] Developer Infrastructure

### CODE REVIEW

File: content/zh/news/releases/1.6.x/announcing-1.6/_index.md [hunk 1]
```diff
@@ -0,0 +1,110 @@
...
+部署在 VM 上的工作负载）的人来说，
+比之前的任何时候都更加简单。我们创建此 API
+是为了在 Istio 中为非 Kubernetes
+工作负载提供一流的表现。
+它将虚拟机或裸金属工作负载提升到与 Kubernetes `Pod`
+相同的级别，而不仅仅是具有 IP 地址的端点。
+您现在甚至可以定义 Pod 和 VM 并存的 Service。
+为什么这是有用的呢？因为您现在可以为同一服务实现混合部署（VM 和 Pod），
+这提供了一种将 VM 工作负载迁移到 Kubernetes
+集群的好方法，而不是通过中断其进出站流量来实现。
```
Comment (windsonsea) [inline content/zh/news/releases/1.6.x/announcing-1.6/_index.md:85]: 有点过度折行了，通常在 col 80 左右折行，也就是每行可以至少放 40 个中文字符或 80 个英文字符

File: content/zh/news/releases/1.6.x/announcing-1.6/_index.md [hunk 2]
```diff
@@ -0,0 +1,110 @@
...
+我们已经完成了这一转变，并将所有功能完全迁移到了 Istiod 中。
+这使我们能够删除 Citadel、Sidecar 注入器和
+Galley 的独立部署。
+
+好消息！我们为使用 Kubernetes 中新的
+alpha 功能的开发人员提供了简化的体验。
+如果您在 Kubernetes [`EndpointPort`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#endpointport-v1beta1-discovery-k8s-io)
+或 [`ServicePort`](https://kubernetes.io/docs/reference/generated/kubernetes-api/v1.18/#serviceport-v1-core)
+API 中使用新的 `appProtocol` 字段（这是 1.18 中的 Alpha 版本功能），
+你将不再需要在您的 `Service` 中附加名称字段以表示协议。
```
Comment (windsonsea) [inline content/zh/news/releases/1.6.x/announcing-1.6/_index.md:39]:
```suggestion
您将不再需要在您的 `Service` 中附加名称字段以表示协议。
```

File: content/zh/news/releases/1.6.x/announcing-1.6/_index.md [hunk 3]
```diff
@@ -0,0 +1,110 @@
...
+
+上一个版本中，我们引入了 **Istiod**，
+这是一个新模块，它通过组合多个服务的功能来减少
+Istio 安装中的组件数量。在 Istio 1.6 中，
+我们已经完成了这一转变，并将所有功能完全迁移到了 Istiod 中。
+这使我们能够删除 Citadel、Sidecar 注入器和
+Galley 的独立部署。
+
+好消息！我们为使用 Kubernetes 中新的
+alpha 功能的开发人员提供了简化的体验。
```
Comment (windsonsea) [inline content/zh/news/releases/1.6.x/announcing-1.6/_index.md:35]:
```suggestion
好消息！我们为使用 Kubernetes 中新的 Alpha 功能的开发人员提供了简化的体验。
```

**Review by windsonsea — COMMENTED** [review]
有些过度折行了，每行 40 个中文或 80 个英文字符，around col 80 左右折行

File: content/zh/news/releases/1.6.x/announcing-1.6/_index.md (hunk 1 above)
Comment (wilsonwu) [reply content/zh/news/releases/1.6.x/announcing-1.6/_index.md:85]: > 有点过度折行了，通常在 col 80 左右折行，也就是每行可以至少放 40 个中文字符或 80 个英文字符

Got it, I will all related probem soon.

Comment (wilsonwu) [reply content/zh/news/releases/1.6.x/announcing-1.6/_index.md]: All fixed.

File: content/zh/news/releases/1.6.x/announcing-1.6/_index.md (hunk 2 above)
Comment (wilsonwu) [reply content/zh/news/releases/1.6.x/announcing-1.6/_index.md]: Fixed in new commit.

**Comment by wilsonwu** [comment]
> 有些过度折行了，每行 40 个中文或 80 个英文字符，around col 80 左右折行

Fixed in the new commit.

**Review by windsonsea — APPROVED** [review]

**Pull request merged**